packages/commands/src/evmDeposit.ts

```
@@ -0,0 +1,125 @@
+import { type NetworksSchema } from "@zetachain/networks/dist/src/types";
+
+import { ZetaChainClient } from "../../client/src/client";
+import { evmDeposit } from "../../client/src/evmDeposit";
+
+const main = async (options: {
+  amount: string;
+  erc20?: string;
+  gateway?: string;
+  receiver: string;
+  revertAddress: string;
+  callOnRevert: boolean;
```

ESLint

[error] Expected interface keys to be in ascending order. 'callOnRevert' should be before 'revertAddress'.

(typescript-sort-keys/interface)